app/representers/api/podcast_representer.rb

```diff
@@ -9,6 +9,9 @@ class Api::PodcastRepresenter < Api::BaseRepresenter
   property :updated_at
   property :published_at
 
+  property :pub_date, writeable: false
+  property :last_build_date, writeable: false
```

Comment:
Is this necessary, since `updated_at` is already represented?  (And this doesn't look at the episodes the way `pub_date` does)